src/index.js

```diff
@@ -92,9 +100,9 @@ class LndEngine {
         //
         // We make this initial call to check if engine is unlocked before continuing
         // to validate the current engine
-        const engineIsUnlocked = await this.isEngineUnlocked()
+        this.unlocked = await this.isEngineUnlocked()
```

Comment:
Nit, `isEngineUnlocked` is a bit redundant.

Reply:
This change is actually outside the scope of this feature. `isEngineUnlocked` was added in a previous PR. Once we create a standard for the engine (arch design) we can revisit (ive added a card to our backlog JIC)

Reply:
I agree that this change is outside the scope of the PR, but I don't understand how `Once we create a standard for the engine (arch design)` is relevant to the redundant naming?

It seems like the pattern we follow is not to have engine in the name of functions? ex. `isAvailable`, `isNodeConfigValid` so `isEngineUnlocked` goes against that pattern.

Reply:
We'll eventually need to publish a spec for an engine, which at that time we can modify the calls. I understand that `engine` isn't in the name of any other function, but its beside the point. I agree that the name can be changed